internal/app/app.go

```diff
@@ -1211,6 +1213,15 @@ func (app *App) performSwitchover(clusterState map[string]*NodeState, activeNode
 	}
 
 	app.logger.Info("switchover: phase 1: stop replication")
+
+	oldMasterNode := app.cluster.Get(oldMaster)
+	if clusterState[oldMaster].PingOk {
+		err := oldMasterNode.StopExternalReplication()
```

Comment:
We need add StopExternalReplication() to repairSlaveNode() method:
https://github.com/yandex/mysync/blob/master/internal/app/app.go#L1649